OOP/oop_home_task.py

```diff
@@ -0,0 +1,475 @@
+"""
+Home task for Module 5 - Classes. OOP.
+
+Create a tool, which will do user generated news feed:
+
+1.User select what data type he wants to add
+
+2.Provide record type required data
+
+3.Record is published on text file in special format
+
+
+
+You need to implement:
+
+1.News – text and city as input. Date is calculated during publishing.
+
+2.Private ad – text and expiration date as input. Day left is calculated during publishing.
+
+3.Your unique one with unique publish rules.
+
+
+
+Each new record should be added to the end of file. Commit file in git for review.
+"""
+
+import datetime
+
+SEPARATOR = '------------------------------\n\n'
+
+
+class Publication:
+    """
+    Class represents publication entity.
+    """
+    def __init__(self, publication_date, publication_type, city, publication_text):
+        """
+        Function is constructor for Publication.
+        @param publication_date: date of publication
+        @param publication_type: type of publication
+        @param city: city where something happened
+        @param publication_text: text of publication
+        """
```

Comment:
Create a documentation is a good practice definitely, but if it's too redundant it's not good. Documentation is needed only for those function/methods/classes where some behavior is not so obvious. In this case, as in other cases where you create docstring for constructors, don't you think that it will be comprehensive even without documentation?

I would rethought where we should add docu, and where we can live without it. In this particular case it's not needed.
Please follow all your code and delete it everywhere where you feel it's not really needed.

Reply:
Fixed.